src/inference/src/os/lin/lin_system_conf.cpp

```diff
@@ -53,6 +62,131 @@ struct CPU {
         if (_cores == 0) {
             _cores = _processors;
         }
+
+        /**
+         * New method to get CPU infomation and CPU map. Below is the structure of CPU map and two sample.
+         *  1. Four processors of two Pcore
+         *  2. Four processors of four Ecores shared L2 cache
+         *
+         *  Proc ID : Socket ID | HW Core ID | Phy Core of Pcores | Logic Core of Pcores | ID of Ecore Group | Used
+         *  (index)
+         *     0         1            1                0                      1                   0             0
+         *     1         1            1                1                      0                   0             0
+         *     2         1            2                0                      2                   0             0
+         *     3         1            2                2                      0                   0             0
+         *     4         1            3                0                      0                   1             0
+         *     5         1            4                0                      0                   1             0
+         *     6         1            5                0                      0                   1             0
+         *     7         1            6                0                      0                   1             0
```

Comment:
1. Why socket ID starts for 1 and not from 0?
2. Why HW core ID starts for 1 and not from 0? 
3. Can we unite "Phy Core of Pcores" and "Logic Core of Pcores" in one column which will just mark the item to 1 if this is logic processor and to 0 if it is physical processor? 
4. Used column is not filled anywhere. Whats its purpose?

Reply:
updated in new commit